osf/models/user.py

```diff
@@ -1283,6 +1283,11 @@ def change_password(self, raw_old_password, raw_new_password, raw_confirm_passwo
             raise ChangePasswordError(issues)
         self.set_password(raw_new_password)
         self.reset_old_password_invalid_attempts()
+        if self.verification_key_v2:
+            self.verification_key_v2['expires'] = timezone.now()
+        # new verification key (v1) for CAS
+        self.verification_key = generate_verification_key(verification_type=None)
+        self.save()
```

Comment:
Is this `.save()` necessary?

Reply:
I can move it back to the view

Reply:
Not a big deal: we just try to avoid implicit saving in the model.

Reply:
gotcha